modules/sitemaps/sitemap-finder.php

```diff
@@ -29,7 +29,8 @@ class Jetpack_Sitemap_Finder {
 	 * @return string Complete URI of the given sitemap file.
 	 */
 	public function construct_sitemap_url( $filename ) {
-		return jetpack_sitemap_uri( $filename );
```

Comment:
This is the only usage of this function. What about modifying the function itself to do what we need it to do?

Reply:
This function is called on WPCOM and changing it would have unknown side-effects there.